ecosystem/typescript/sdk_v2/src/crypto/secp256k1.ts

```diff
@@ -161,11 +161,11 @@ export class Secp256k1PrivateKey extends PrivateKey {
 
   /**
    * Derive the Secp256k1PublicKey from this private key.
-   * 
+   *
    * @returns Secp256k1PublicKey
    */
   publicKey(): Secp256k1PublicKey {
-    const bytes = secp256k1.getPublicKey(this.key.toUint8Array());
+    const bytes = secp256k1.getPublicKey(this.key.toUint8Array(), false);
```

Comment:
why `false`?

Reply:
Good question. If true, the secp256k1 lib will compress the public key into 32 bytes. Instead we want an un-compressed one, which is 65 bytes.